android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/annotation/core/annotations/AnnotationManager.kt

```diff
@@ -0,0 +1,122 @@
+package com.github.barteksc.pdfviewer.annotation.core.annotations
+
+import android.graphics.PointF
+import com.github.barteksc.pdfviewer.annotation.core.shapes.Relations
+import com.github.barteksc.pdfviewer.util.PublicValue
+import com.github.barteksc.pdfviewer.util.logError
+import com.lowagie.text.Rectangle
+import com.lowagie.text.pdf.PdfAnnotation
+import com.lowagie.text.pdf.PdfArray
+import com.lowagie.text.pdf.PdfDictionary
+import com.lowagie.text.pdf.PdfLayer
+import com.lowagie.text.pdf.PdfName
+import com.lowagie.text.pdf.PdfNumber
+import com.lowagie.text.pdf.PdfReader
+import com.lowagie.text.pdf.PdfStamper
+import com.lowagie.text.pdf.PdfString
+import java.awt.Color
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.FileOutputStream
+import java.io.InputStream
+import java.util.UUID
+
+object AnnotationManager {
+    private val TAG = AnnotationManager.javaClass.simpleName
+
+    /** Draws a layer with a rectangle annotation to a PDF document with 1 page */
+    @JvmStatic
+    fun addRectangleAnnotation(
+        rectCorners: List<PointF>,
+        file: File,
+        relations: Relations? = null
+    ): Boolean {
+        if (!file.exists()) throw FileNotFoundException()
```

Comment:
Method signature doesnt mention it can throw and we are using try/catch a few lines below so I suggest to remove this so it can be catched.